cmake/Modules/FindMPLIB.cmake

```diff
@@ -1,7 +1,10 @@
+message(STATUS "Trying to find GMP or MPIR.")
+message(STATUS "Failure to find either is not a problem. Only one of them is required.")
```

Comment:
What % of users are going to be using GMP vs MPIR? Probably a significant portion; to manage verbosity this could be moved into the else(GMP_FOUND) conditional. The "other two new messages above/below are possibly not necessary.

Reply:
Done that.